### PR TITLE
Hyperlink contact details

### PR DESCRIPTION
**Pull request by richdouglasevans**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -58,7 +58,8 @@ <h2><a href="#{{slug}}"><span class="date">{{short_date date}}</span> {{name}}</
 
     <div id="content">
 
-      <h1>Summer of Hacks is an annual series of friendly, practical &amp; free events organised by the Oxford tech community.</h1>
+      <h1>Summer of Hacks is an annual series of friendly, practical, &amp; free
+        events in Oxford organised by the local tech community.</h1>
```
Comment (richdouglasevans) [inline src/index.html:62]: Emphasized that these events are held in Oxford 😄

File: src/index.html [hunk 2]
```diff
@@ -109,14 +110,18 @@ <h2>&rarr; Attend an event</h2>
       <p>The best way to get involved is to come along to one of our events.</p>
 
       <h2>&rarr; Sponsor Us</h2>
-      <p>To keep our events free, we depend on sponsors to fund food, drink and venue hire.  If you are able to help out - we'd really love to hear from you.</p>
+      <p>To keep our events free we depend on sponsors to fund food, drink and venue hire.
+        If you are able to help out we'd really love to hear from you:
+        <a href="https://twitter.com/Ox_SOH">Tweet</a> or
+        <a href="mailto:ox.summer.of.hacks@gmail.com">email</a> us.</p>
```
Comment (richdouglasevans) [inline src/index.html:116]: Provided ways for potential sponsors to contact us.

> We should open our Twitter to public DMs.

File: src/index.html [hunk 3]
```diff
@@ -109,14 +110,18 @@ <h2>&rarr; Attend an event</h2>
...
 
       <h2>&rarr; Slack</h2>
-      <p>We discuss Summer of Hacks on the <a href="slack.digitaloxford.com">digital oxford slack</a>. Come and say hi on the #soh channel.</p>
+      <p>We discuss Summer of Hacks on the <a href="slack.digitaloxford.com">Digital Oxford Slack</a>.
+        Come and say hi on the [#soh](https://digitaloxford.slack.com/messages/C17K459ND/) channel.</p>
```
Comment (richdouglasevans) [inline src/index.html:120]: Linked directly to the `#soh` channel so that folk can surf to it directly.

File: src/index.html [hunk 4]
```diff
@@ -109,14 +110,18 @@ <h2>&rarr; Attend an event</h2>
...
 
       <h2>&rarr; Twitter</h2>
       <p>Some short and sweet tweets will be served from
-      <a href="https://twitter.com/Ox_SOH">our twitter account</a>.
+      <a href="https://twitter.com/Ox_SOH">our Twitter account</a>.
```
Comment (richdouglasevans) [inline src/index.html:124]: Twitter is an app name so I've capitalised it.

> Did the same for mention of Slack a few lines up too.

File: src/index.html (hunk 2 above)
Comment (lpmi-13) [reply src/index.html:116]: public DMs opened.

File: src/index.html (hunk 4 above)
Comment (lpmi-13) [reply src/index.html:124]: clap

**Pull request merged**